psi4/src/psi4/fnocc/ccsd.cc

```diff
@@ -39,8 +39,27 @@
 #include "psi4/libmints/basisset.h"
 #include "psi4/lib3index/3index.h"
 
+#include <ctime>
```

Comment:
We seem to be carrying around this duplication. Can we move this to a its own header somewhere? From this PR and others this winds up in 6-8 places.

Reply:
OK! I can move to `psi4/include/psi4/times.h`